extension/chrome/elements/compose.ts

```diff
@@ -106,9 +106,19 @@ Catch.try(async () => {
   }
   const processedUrlParams = { acctEmail, draftId, threadId, ...replyParams, frameId, tabId, isReplyBox, skipClickPrompt, parentTabId, disableDraftSaving, debug };
   const storageGetKey = async (senderEmail: string): Promise<KeyInfo> => {
-    const [primaryKi] = await Store.keysGet(acctEmail, ['primary']);
-    Assert.abortAndRenderErrorIfKeyinfoEmpty(primaryKi);
-    return primaryKi;
+    let result: KeyInfo | undefined;
+    const keys = await Store.keysGet(senderEmail);
```

Comment:
If it worked the way it was written now, than there is an error elsewhere, and that should be fixed. `Store.keysGet('flowcrypt.compatibility@gmail.com');` should always return nothing.

Reply:
Yes, it's working, and yes I understand that it shouldn't work. Going to investigate it and fix.

Reply:
I found the reason: when saving/getting elements in localstorage  we remove all characters from the email address except numbers and letters:
```TS
 static singleScopeRawIndex = (scope: string, key: string) => `cryptup_${scope.replace(/[^A-Za-z0-9]+/g, '').toLowerCase()}_${key}`;
```
Emails `flowcrypt.compatibility@gmail.com` and `flowcryptcompatibility@gmail.com` are exactly the same but the second one doesn't contain '.'.
So because of it, both emails return me some data from localstorage.